backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -265,6 +271,8 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
   /** If there are no responses pending this behaves like `failAndStop`, otherwise this goes to `FailedState` and waits
     * for all the pending responses to come back before stopping. */
   def failAndAwaitPendingResponses(failure: CacheCopyError, command: IoCommand[_], data: StandardCacheHitCopyingActorData): State = {
+    standardParams.blacklistCache foreach { _.blacklistHit(standardParams.cacheHit) }
```

Comment:
I wonder whether it's a good idea to write out metrics for the blacklistCache addition rate as well as the lookup rate? Might help to give us a better idea on how big we need to make them?

Reply:
yup agree